content/english/javascript-basics/Arrays/Iterating over Arrays.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Iterating over Arrays"
```

Comment:
Can you rename all the files so that they use lowercase, and that they don't use any spaces in the file names?

e.g. `content/english/javascript-basics/arrays/iterating_over_arrays.md`